### PR TITLE
Remove unused deck upgrade and deck download code

### PR DESCRIPTION
**Pull request by hssm**
Since neither feature is part of AnkiDroid anymore, and I like deleting things, here's a big red commit that should make things a little bit lighter.

I used lint to verify the strings weren't being used before removing them. If there are no objections, I might go ahead and remove every other unused string in the application in another PR. They number in 9k+ range when you include all the translations, so not only will removing them reduce the apk size, it should also save time for translators (or rather, we should stop wasting their time with unused strings).

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/Info.java [hunk 1]
```diff
@@ -234,327 +172,6 @@ public void onClick(View v) {
...
-                }
-                mWebView.setWebViewClient(new MobileAnkiWebview());
-                mWebView.loadUrl(res.getString(R.string.shared_decks_url));
-                mWebView.getSettings().setJavaScriptEnabled(true);
-                termsAndConditionsView.setVisibility(View.VISIBLE);
-                continueButton.setText(res.getString(R.string.download_button_return));
-                break;
-
-            case TYPE_UPGRADE_DECKS:
-                setTitle(R.string.deck_upgrade_title);
```
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/Info.java]: If we're going to remove the upgrade wizard, I think we need to at least leave some kind of instructions in for users upgrading from Anki1. For example we only just recently upgraded the version in the Amazon store from 1.x to 2.4.

Maybe we could have a dialog linking to [here](https://code.google.com/p/ankidroid/wiki/Upgrading) popup everytime the DeckPicker is opened with an empty collection and *.anki files in the AnkiDroid directory.

Comment (hssm) [inline AnkiDroid/src/main/java/com/ichi2/anki/Info.java]: I believe we've already committed to not supporting upgrades anymore. You removed the last way to access the wizard [here](https://github.com/ankidroid/Anki-Android/pull/649), and I made this PR with that in mind.

The code I'm removing is completely inaccessible from anywhere in the application. No functionality is being removed here.

Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/Info.java]: Ah that's right I had forgotten about the crashes

**Comment by timrae** [comment]
Grepping for "Info.class" I can see that the Info activity only uses `TYPE_ABOUT` and `TYPE_NEW_VERSION`, so I think you can remove all that sync crap as well

**Comment by hssm** [comment]
Yeah you're right, I've gone ahead and done that. I fixed up a couple warnings while I was at it, and removed a line that writes an *sslAcceptAll* preference value but isn't used anywhere else in the application.

I think this class originally started out as a generic "show a html page" class but lots of stuff got moved into it. I don't like that the `lastVersion` preference is set in here.

**Pull request merged**